interactions/models/discord/guild.py

```diff
@@ -2263,7 +2263,7 @@ def get_channels(self) -> List["models.TYPE_VOICE_CHANNEL"]:
         """
         return [self._client.get_channel(channel_id) for channel_id in self._channel_ids]
 
-    async def fetch_channels(self, *, force: bool = False) -> List["models.TYPE_VOICE_CHANNEL"]:
+    async def fetch_channels(self, *, force: bool = False) -> List["models.TYPE_GUILD_CHANNEL"]:
```

Comment:
This is meant to be `TYPE_VOICE_CHANNEL` due to it being a `GuildWidget`, so this should be reverted.

Reply:
This change is reverted